util/font_8x8.c

```diff
@@ -0,0 +1,175 @@
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x5f, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x03, 0x00, 0x03, 0x00, 0x00, 0x00},
+	{0x64, 0x3c, 0x26, 0x64, 0x3c, 0x26, 0x24, 0x00},
+	{0x26, 0x49, 0x49, 0x7f, 0x49, 0x49, 0x32, 0x00},
+	{0x42, 0x25, 0x12, 0x08, 0x24, 0x52, 0x21, 0x00},
+	{0x20, 0x50, 0x4e, 0x55, 0x22, 0x58, 0x28, 0x00},
+	{0x00, 0x00, 0x00, 0x03, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x1c, 0x22, 0x41, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x41, 0x22, 0x1c, 0x00, 0x00},
+	{0x00, 0x15, 0x15, 0x0e, 0x0e, 0x15, 0x15, 0x00},
+	{0x00, 0x08, 0x08, 0x3e, 0x08, 0x08, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x50, 0x30, 0x00, 0x00, 0x00},
+	{0x00, 0x08, 0x08, 0x08, 0x08, 0x08, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x40, 0x00, 0x00, 0x00, 0x00},
+	{0x40, 0x20, 0x10, 0x08, 0x04, 0x02, 0x01, 0x00},
+	{0x00, 0x3e, 0x41, 0x41, 0x41, 0x3e, 0x00, 0x00},
+	{0x00, 0x00, 0x41, 0x7f, 0x40, 0x00, 0x00, 0x00},
+	{0x00, 0x42, 0x61, 0x51, 0x49, 0x6e, 0x00, 0x00},
+	{0x00, 0x22, 0x41, 0x49, 0x49, 0x36, 0x00, 0x00},
+	{0x00, 0x18, 0x14, 0x12, 0x7f, 0x10, 0x00, 0x00},
+	{0x00, 0x27, 0x49, 0x49, 0x49, 0x71, 0x00, 0x00},
+	{0x00, 0x3c, 0x4a, 0x49, 0x48, 0x70, 0x00, 0x00},
+	{0x00, 0x43, 0x21, 0x11, 0x0d, 0x03, 0x00, 0x00},
+	{0x00, 0x36, 0x49, 0x49, 0x49, 0x36, 0x00, 0x00},
+	{0x00, 0x06, 0x09, 0x49, 0x29, 0x1e, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x12, 0x00, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x00, 0x52, 0x30, 0x00, 0x00, 0x00},
+	{0x00, 0x00, 0x08, 0x14, 0x14, 0x22, 0x00, 0x00},
+	{0x00, 0x14, 0x14, 0x14, 0x14, 0x14, 0x14, 0x00},
+	{0x00, 0x00, 0x22, 0x14, 0x14, 0x08, 0x00, 0x00},
+	{0x00, 0x02, 0x01, 0x59, 0x05, 0x02, 0x00, 0x00},
+	{0x3e, 0x41, 0x5d, 0x55, 0x4d, 0x51, 0x2e, 0x00},
+	{0x40, 0x7c, 0x4a, 0x09, 0x4a, 0x7c, 0x40, 0x00},//A
```

Comment:
the comments are still here, let's either comment all or none, I'd opt for removing them